virttest/bootstrap.py

```diff
@@ -22,6 +22,7 @@
 
 
 LOG = logging.getLogger("avocado.app")
+qemu_headers = ['types.h', 'socket.h', 'unistd.h']
```

Comment:
I don't understand the benefit of defining the qemu_headers in top level. Wouldn't it suffice to simply update the list on line 49? If not that the name should be at least UPPER_CASE to conform with python style guidelines